server/pages/index.jade

```diff
@@ -54,6 +54,8 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 				link(rel='stylesheet', href=urls['style-debug.css'])
 			else
 				link(rel='stylesheet', href=urls['style.css'])
+		style(type='text/css' id='critical-path-css')
+			!= Array.from(css).join('')
```

Comment:
I meant, we should do it when we implement SSR since it's only really needed in that case. Ok to keep it now if you prefer though